### ISSUE TITLE
engineLoadIMG fails to load large archive

### ISSUE THREAD

**Issue by nekjutsu** [multitheftauto/mtasa-blue#3701]
### Describe the bug

engineLoadIMG returns false when trying to load a large archive (>= 1 GB)

### Steps to reproduce

1. Add .img file to meta.xml
2. Call engineLoadIMG

### Version

Client: v1.6.0-22702 (nightly)
Server: v1.6.0-22702 (nightly, win32)

### Security Policy

- [x] I have read and understood the Security Policy and this issue is not security related.

**Comment by Xenius97** [comment]
Default GTA:SA has 2GB limit for IMG Archives (by default? or with limit adjuster? - idk no documentation)
It should work but maybe MTA has different limitations.

**Comment by nekjutsu** [comment]
> Default GTA:SA has 2GB limit for IMG Archives (by default? or with limit adjuster? - idk no documentation) It should work but maybe MTA has different limitations.

As i know default GTA:SA limit is exactly 1GB
In MTA i tested 1.5GB

**Comment by TheNormalnij** [comment]
Yeah, It's a MTA limitation. The HTTP server doesn't handle files bigger than 1GB
